tools/src/main/java/org/apache/kafka/tools/StreamsResetter.java

```diff
@@ -687,7 +687,7 @@ public List<String> inputTopicsOption() {
         }
 
         public List<String> intermediateTopicsOption() {
-            log.warn("intermediateTopicsOption will be removed in version 5.0");
+            log.warn("intermediateTopicsOption is deprecated and will be removed in version 5.0");
```

Comment:
This is the warning message it will log, when the option is used

Reply:
We don't know for sure if we remove it in 5.0 (depends on timeline). Would only say "will be removed in a future release."

Reply:
Seems you did not address this comment yet?

Reply:
Btw: Given that this is a command line tool, I think we should not actually user a `Logger` but `System.out.println(...)` here (we do this everywhere else, too).